psga/config/pipeline-schema.yaml

```diff
@@ -0,0 +1,95 @@
+pipeline:
+  name: sars-cov-2
+  docker_image: ''
+  protocols:
+  - name: illumina-sars-cov-2-artic-v3
+    kit:
+      name: Artic V3
+      slug: articv3
+    sequencer_technology:
+      name: Illumina
+      slug: illumina
+    pathogen:
+      name: Sars-CoV-2
+      slug: sars-cov-2
+  - name: ont-sars-cov-2-artic-v4
+    kit:
+      name: Artic V4
+      slug: articv4
+    sequencer_technology:
+      name: ONT
+      slug: ont
+    pathogen:
+      name: Sars-CoV-2
+      slug: sars-cov-2
+
+pipeline_version:
+  docker_tag: 1.0.1
+
+input_fields:
```

Comment:
@pdp10 We also need help defining the input and output fields that the pipeline expects to receive and output.

Reply:
Input
* `sample_id` is a GUID. 
* I believe what here you call `batch` should be `run`. That's the parameter the pipeline uses. It says that that's a number. I would make that a GUID too.
* `study_type`, `score` and `collection_date` are not used by the pipeline

Reply:
some of these field names will need to map to the BE and potentially the lab csv file.
I believe https://confluence.congenica.net/display/PSG/PSGA+Pipeline+Architecture may guide you - @Aiky30 can you also cross check with the BE input serializer and see if there any discrepancies in field naming? Thanks